src/utils.ts

```diff
@@ -8,3 +10,13 @@ export function insertIf (condition: boolean, element: any): any[] {
 export function dateToday () {
   return formatDate((new Date()).toISOString(), DATE_FORMATS.date_value);
 }
+
+export function getPercentValue (value?: null | string): string {
+  if (typeof value === 'undefined' || value === null || value === '') { return ''; }
+  return new Decimal(value).div(CENT_DECIMAL).toString();
+}
+
+export function getPercentDisplay (value?: null | string): string {
+  if (typeof value === 'undefined' || value === null || value === '') { return ''; }
+  return new Decimal(value).times(CENT_DECIMAL).toString();
```

Comment:
Shouldn't this one be a number (based on the functions above)? Although the name is misleading if that's the case

Reply:
Yeah the original functions returned numbers and it caused the disastrous results seen in the second screenshot :stuck_out_tongue: We really need to pass decimal values around as either string or Decimal, and since HTTP and editing require string, that seemed like the best option for here.

Reply:
gotcha 👍